continuous-improvement/monitoring/feedback.html

```diff
@@ -6,385 +6,13 @@
       <meta charset="utf-8">
       <!-- Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
          wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html -->
-      <title>Page feedback tool - Continuous improvement of the Government of Canada web content - Canada.ca</title>
-      <meta content="width=device-width,initial-scale=1" name="viewport">
-      <link href="https://www.canada.ca/etc/designs/canada/wet-boew/assets/favicon.ico" rel="icon" type="image/x-icon">
-      <link rel="stylesheet" href="https://www.canada.ca/etc/designs/canada/wet-boew/css/theme.min.css">
-      <link rel="stylesheet" href="../../css/custom.css">
-      <link rel="stylesheet" href="../../css/provisional-alerts.css">
-      <link rel="stylesheet" href="../../css/split-h1.css">
-      <link href="https://test.canada.ca/covid-19-guidance/proto/css/alpha-beta-banner.css" rel="stylesheet">
-      <noscript>
-         <link rel="stylesheet" href="https://www.canada.ca/etc/designs/canada/wet-boew/wet-boew/css/noscript.min.css" />
-      </noscript>
-      <style>
-         .pattern-demo {
-         padding: 19px;
-         }
-      </style>
 
-      <!-- Global site tag (gtag.js) - Google Analytics -->
-      <script async src="https://www.googletagmanager.com/gtag/js?id=UA-105628416-2"></script>
-      <script>
-         window.dataLayer = window.dataLayer || [];
-         function gtag() { dataLayer.push(arguments); }
-         gtag('js', new Date());
+      <meta http-equiv="Refresh" content="0; URL=https://design.canada.ca/feedback/index.html">
```

Comment:
Let's talk to @prycrane about how to manage redirects properly. We have a _redirect file that should host all of our redirects.